components/x-podcast-launchers/readme.md

```diff
@@ -64,7 +64,7 @@ Feature              | Type     | Required | Notes
 ---------------------|----------|----------|------------------
 `acastRSSHost`       | String   | Yes      | e.g. 'https://acast.access.com'
 `conceptId`          | String   | Yes      |
-`conceptName`        | String   | Yes      | x-follow-button prop.
-`isFollowed`         | Boolean  | Yes      | x-follow-button prop. Whether the podcast concept id is followed or not.
-`csrfToken`          | String   | Yes      | x-follow-button prop. Value included in a hidden form field.
-`renderFollowButton` | Function | No       | x-follow-button prop. Optional render prop.
+`renderFollowButton` | Function | No       | Optional render prop for the follow button
+
+`conceptName`, `isFollowed` and `csrfToken` are required for
```

Comment:
I think this could be kept more 'evergreen' by staying generic. For example:

> Additional props such as the conceptId may be required by x-follow-button. Documentation for these is available over in the component's readme.